src/Action/Shop/DownloadCreditMemoAction.php

```diff
@@ -22,32 +22,28 @@
 
 final class DownloadCreditMemoAction
 {
-    private CreditMemoPdfFileGeneratorInterface $creditMemoPdfFileGenerator;
-
-    private CreditMemoCustomerRelationCheckerInterface $creditMemoCustomerRelationChecker;
-
-    private CreditMemoFileResponseBuilderInterface $creditMemoFileResponseBuilder;
-
     public function __construct(
-        CreditMemoPdfFileGeneratorInterface $creditMemoPdfFileGenerator,
-        CreditMemoCustomerRelationCheckerInterface $creditMemoCustomerRelationChecker,
-        CreditMemoFileResponseBuilderInterface $creditMemoFileResponseBuilder
+        private CreditMemoPdfFileGeneratorInterface $creditMemoPdfFileGenerator,
+        private CreditMemoCustomerRelationCheckerInterface $creditMemoCustomerRelationChecker,
+        private CreditMemoFileResponseBuilderInterface $creditMemoFileResponseBuilder,
+        private bool $hasEnabledPdfFileGenerator = true
```

Comment:
Do we need this default value? we have the parameter `enabled` set as true by default in configuration